### DISCUSSION TITLE
No menue option for add metabolizing enzyme... (Database Query)

### DISCUSSION THREAD

**Discussion by urban-blade** [Open-Systems-Pharmacology/Forum#1634] [Questions & Problems]
![pk sim](https://github.com/Open-Systems-Pharmacology/Forum/assets/79349482/03c720e5-dbee-45db-b869-d7b2e144840a)

Hi Team,

No _add metabolizing enzyme (database query)_ was shown when right click the context menu _metabolizing enzyme_.
How can i resolve it?

**IMPORTANT: Issues are now obsolete**
Starting May 7th, 2021, the OSP issues will not be used anymore for discussions. Instead, the Discussions feature from Github will be used

Please see the announcement here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/894

To start a discussion, simply go to 
https://github.com/Open-Systems-Pharmacology/Forum/discussions and click on New Discussion

**Comment by tobiasK2001** [comment]
did you add the path to the gene-expression database as described here: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options?

Best, Tobias
